Source/LocationInfo.swift

```diff
@@ -0,0 +1,21 @@
+public struct LocationInfo {
+    //    #namespace     String   The name of the namespace / module
+    //    #file          String   The name of the file in which it appears.
+    //    #line          Int      The line number on which it appears.
+    //    #column        Int      The column number in which it begins.
+    //    #function      String   The name of the declaration in which it appears.
+    //    #dsohandle     String   The dso handle.
+    public var file : String
+    public var line : Int
+    public var column : Int
+    public var function : String
+
+    init (file: String, line: Int, column: Int, function: String) {
```

Comment:
Two (nitpicky) things:
- These variables being `var`s is kind of useless, since it's already a struct and can't be mutated without making the struct itself a var.
- structs have default member wise initializers that look exactly like this, so the whole initializer can be removed :)